library/EasyPipeline/stack.yaml.lock

```diff
@@ -3,262 +3,273 @@
 # For more information, please see the documentation at:
 #   https://docs.haskellstack.org/en/stable/lock_files
 
-snapshots:
```

Comment:
Are the `stack.yaml.lock` changes related to the rest of the PR?

Reply:
A guess: maybe it's that the `import Debug.Pretty.Simple ( pTraceShowM )` requires package `pretty-compact` and that adding that caused `stack` to rearrange the `stack.yaml.lock` file for some reason?

If so, would you mind splitting the change to add that package into a separate commit (or PR) with an explanation of that, so that it's clear that the new functionality itself doesn't depend on package changes (only a temporary trace message does)?